File: sub-mysql-web-service/src/main/java/org/github/mysql/web/base/service/impl/SysUserServiceImpl.java
package org.github.mysql.web.base.service.impl;

import org.github.mysql.web.base.entity.SysUserEntity;
import org.github.mysql.web.base.mapper.ISysUserMapper;
import org.github.mysql.web.base.service.ISysUserService;
import com.baomidou.mybatisplus.extension.service.impl.ServiceImpl;
import org.springframework.stereotype.Service;

/**
 * <p>
 * 系统用户 服务实现类
 * </p>
 *
 * @author JYD_XL
 * @since 2021-05-22
 */
@Service
public class SysUserServiceImpl extends ServiceImpl<ISysUserMapper, SysUserEntity> implements ISysUserService {}

File: sub-mysql-mydb-service/src/main/java/org/github/mysql/web/base/service/impl/SysUserServiceImpl.kt
@file:Suppress("RedundantSemicolon")

package org.github.mysql.web.base.service.impl;

import org.github.mysql.web.base.entity.SysUserEntity;
import org.github.mysql.web.base.mapper.ISysUserMapper;
import org.github.mysql.web.base.service.ISysUserService;
import com.baomidou.mybatisplus.extension.service.impl.ServiceImpl;
import org.springframework.stereotype.Service;

/**
 * 系统用户 服务实现类
 *
 * @author JYD_XL
 */
@Service
open class SysUserServiceImpl : ServiceImpl<ISysUserMapper, SysUserEntity>(), ISysUserService
